.github/PULL_REQUESTS/bug_fix_template.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Bug fix
+about: For PermaplanT bug fixes
+title: ''
+labels: 'bug'
+assignees: ''
+---
+
+**Description**
+
+<!-- A description of the bug and the fix. Also link the issue if it exists. -->
```

Comment:
The link should be in the ChangeLog. (And the issue should exist, so "if it exists" can be removed.)